packages/@adobe/spectrum-css-temp/components/stepper/skin.css

```diff
@@ -245,3 +245,34 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-Stepper {
+    --spectrum-textfield-text-color-disabled: GrayText;
+    --spectrum-textfield-border-color-disabled: GrayText;
+    --spectrum-textfield-quiet-border-color-disabled: GrayText;
+    --spectrum-textfield-quiet-border-color-key-focus: Highlight;
+    --spectrum-textfield-border-color-key-focus: Highlight;
+    &:not(.spectrum-Stepper--isQuiet) {
+      &.is-focused, &:focus {
+        outline: 3px solid Highlight;
+      }    
+    }
+    &.spectrum-Stepper--isQuiet {
+      &.is-focused, &:focus {
+        border-width: 2px;
+      }
+      &.focus-ring, &:focus-ring {
+        forced-color-adjust: none;
+        box-shadow: 0 1px 0 0 var(--spectrum-textfield-quiet-border-color-key-focus);
+        .spectrum-Stepper-input {
+          border-color: var(--spectrum-textfield-quiet-border-color-key-focus);
+        }
+        .spectrum-Stepper-button {
+          border-color: var(--spectrum-textfield-quiet-border-color-key-focus);
+        }
+      }
+    }
+   
+
+  }
```

Comment:
Quiet disabled up and down icons should possibly have the disabled color as well?